test/Examples.hs

```diff
@@ -72,20 +75,73 @@ buildExamples session lang tsDir = do
   knownFailures <- knownFailuresForPath tsDir (languageKnownFailuresTxt lang)
   files <- globDir1 (compile ("**/*" <> languageExtension lang)) (Path.toString (tsDir </> languageExampleDir lang))
   let paths = Path.relFile <$> files
-  trees <- forConcurrently paths $ \file -> pure $ HUnit.testCase (Path.toString file) $ do
-    res <- runTask session (runParse (parseFilePath file))
-    case res of
-      Left (SomeException e) -> case cast e of
-        -- We have a number of known assignment timeouts, consider these pending specs instead of failing the build.
-        Just AssignmentTimedOut -> pure ()
-        Just ParserTimedOut     ->     pure ()
-        -- Other exceptions are true failures
-        _                       -> HUnit.assertFailure (show (displayException e))
-      _ -> if file `elem` knownFailures
-              then pure ()
-              else (isRight res) HUnit.@? ("Error: " <> either show show res)
+  trees <- forConcurrently paths $ \file -> do
```

Comment:
This was already the case, but I don’t think this `forConcurrently` is doing anything much—we’re returning `pure` immediately, so there aren’t any actions to run concurrently and then sequence.